### PR TITLE
disable  subscription_management test

### PR DESCRIPTION
**Pull request by AlvaroVega**
fix for: 

```
   1) Subscription management
       When the iotagent stops
         should cease sending measures to the CB:
     Error: Timeout of 5000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/home/runner/work/iotagent-json/iotagent-json/test/unit/ngsiv2/_subscription-management_test.js)
      at listOnTimeout (node:internal/timers:559:17)
      at processTimers (node:internal/timers:502:7)
```

### CODE REVIEW

File: test/unit/ngsiv2/_subscription-management_test.js [hunk 1]
```diff
@@ -92,45 +92,46 @@ describe('Subscription management', function () {
...
+    //             .reply(204);
+    //     });
+
+    //     it('should cease sending measures to the CB', function (done) {
+    //         async.series(
+    //             [
+    //                 async.apply(request, provisionOptions),
+    //                 sendMeasures('32', '87'),
+    //                 waitForMqttRelay(50),
+    //                 iotaJson.stop,
```
Comment (fgalan) [inline test/unit/ngsiv2/_subscription-management_test.js:123]: What about introducing a delay between the stop and the sending of 53 and 1? Maybe that way the test could be stabilised?

Comment (AlvaroVega) [reply test/unit/ngsiv2/_subscription-management_test.js:123]: I've also tried that but it doesn't work.

File: test/unit/ngsiv2/_subscription-management_test.js [hunk 2]
```diff
@@ -92,45 +92,46 @@ describe('Subscription management', function () {
...
-                ],
-                function (error, results) {
-                    should.not.exist(error);
-                    contextBrokerMock.isDone().should.equal(false);
-                    done();
-                }
-            );
-        });
-    });
+    // FIXME: following test is not working as expected
```
Comment (fgalan) [inline test/unit/ngsiv2/_subscription-management_test.js:95]:
```suggestion
    // FIXME: following test is not working as expected. The second mock expectations (the one with
    // alternativeUpdate.json, which includes the NGSIv2 update for 53 and 1 measures) is not correct,
    // as the IOTA is stopedd with iotaJson.stop before doing sendMeasures('53', '1'). Surprisingly, removing
    // that expectation doesn't work either...
```

File: CHANGES_NEXT_RELEASE [hunk 3]
```diff
@@ -0,0 +1 @@
+blabla
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]:
```suggestion
```

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**